sql-db/vertx-sql/src/main/java/io/quarkus/qe/vertx/sql/PoolApplication.java

```diff
@@ -29,7 +29,7 @@
                 .transform(iterator -> iterator.hasNext() ? iterator.next().getLong("active_con") : null);
     }
 
-    @Route(methods = Route.HttpMethod.PUT, path = "/connect")
+    @Route(methods = Route.HttpMethod.GET, path = "/connect")
```

Comment:
We change state of the object with that request. It's not recommended to use GET requests for that

Reply:
We are not changing the state of the application, but just doing two queries: one to get the current timestamp, another one to get the number of active connections.

Reply:
Yes, and "getting current timestamp" is also "adding another connection to count".